bib/cmd/bootc-image-builder/partition_tables.go

```diff
@@ -118,4 +118,39 @@ var partitionTables = distro.BasePartitionTableMap{
 			},
 		},
 	},
+        arch.ARCH_PPC64LE.String(): disk.PartitionTable{
+		UUID: "0x14fc63d2",
```

Comment:
Hmm, I think that UUID format only makes sense for dos types...although, this has me wondering where the UUID referenced in the x86_64 version `D209C89E-EA5E-4FBD-B161-B461CCE297E0` came from?  Maybe we should use that?

Reply:
Thanks. I have pushed an updated commit to use this UUID as well as use disk.PRePartitionGUID and other constants instead of hardcoding the Type values.